docs/src/pages/components/theme.mdx

```diff
@@ -0,0 +1,56 @@
+---
+name: Theme
+route: /components/theme
+---
+
+import Theme from '@pluralsight/ps-design-system-theme'
+
+import { Types, Usage } from '../../components/component'
+
+# Theme
+
+## Usage pattern
+
+To theme a component or part of the page, surround it in the `Theme` component.
+If you're on React < 16, you'll need to wrap a single child, so a child `div`
+may be required.
+
+```typescript noRender startExpanded
```

Comment:
Why `startExpanded`?  It's long.  What makes this one special?

Reply:
there isn't a preview and it's basically the only thing page. seems like this codeblock is the most important thing going on so i didn't want to hide it